ci/test/00_setup_env_native_tsan.sh

```diff
@@ -7,8 +7,8 @@
 export LC_ALL=C.UTF-8
 
 export CONTAINER_NAME=ci_native_tsan
-export DOCKER_NAME_TAG=ubuntu:22.04
-export PACKAGES="clang-13 llvm-13 libc++abi-13-dev libc++-13-dev python3-zmq"
-export DEP_OPTS="CC=clang-13 CXX='clang++-13 -stdlib=libc++'"
+export DOCKER_NAME_TAG=debian:bookworm  # For clang-15
+export PACKAGES="clang-15 llvm-15 libc++abi-15-dev libc++-15-dev python3-zmq"
+export DEP_OPTS="CC=clang-15 CXX='clang++-15 -stdlib=libc++' NO_QR=1"  # qr disabled due to libqrencode 3.4.4 compile failure, https://github.com/bitcoin/bitcoin/pull/26768#issuecomment-1367403430
```

Comment:
FWIW, bumping `qrencode` package version doesn't help. Maybe
```diff
--- a/depends/packages/qrencode.mk
+++ b/depends/packages/qrencode.mk
@@ -10,6 +10,7 @@ $(package)_config_opts += --disable-gprof --disable-gcov --disable-mudflap
 $(package)_config_opts += --disable-dependency-tracking --enable-option-checking
 $(package)_config_opts_linux=--with-pic
 $(package)_config_opts_android=--with-pic
+$(package)_cflags += -Wno-error=int-conversion
 endef
 
 define $(package)_preprocess_cmds
```
and drop `NO_QR=1`?

Reply:
Not sure. I'd prefer to keep build system changes separate from CI-only changes.